Examples/Framework/include/ActsExamples/EventData/MuonSimHit.hpp

```diff
@@ -59,8 +60,8 @@ muonMdtIdentifierFields splitId(Acts::GeometryIdentifier::Value theID) {
 Acts::GeometryIdentifier::Value compressId(muonMdtIdentifierFields f) {
```

Comment:
I just saw that `muonMdtIdentifierFields` is a lower case type. Could you fix that @dimitra97 ?